pages/common/stripe.md

```diff
@@ -0,0 +1,24 @@
+# stripe
+
+> Interact with your Stripe account.
+> More information: <https://github.com/stripe/stripe-cli>.
+
+- Log all activity on your account:
+
+`stripe logs tail`
+
+- Listen for webhook events for `charge.succeeded` and forward them to localhost:3000/events:
+
+`stripe listen --events="charge.succeeded" --forward-to="localhost:3000/events"`
+
+- Send a test webhook event:
+
+`stripe trigger charge.succeeded`
```

Comment:
`charge.succeeded` can be replace by other option, right? So, wrap it with `{{}}`.